diff --git a/node_modules/ink-testing-library/build/index.js b/node_modules/ink-testing-library/build/index.js
index bbdc07b..365971b 100644
--- a/node_modules/ink-testing-library/build/index.js
+++ b/node_modules/ink-testing-library/build/index.js
@@ -68,23 +68,38 @@ class Stderr extends EventEmitter {
     }
 }
 class Stdin extends EventEmitter {
-    constructor() {
+    constructor(options = {}) {
         super(...arguments);
         // eslint-disable-next-line @typescript-eslint/naming-convention
         Object.defineProperty(this, "isTTY", {
             enumerable: true,
             configurable: true,
             writable: true,
-            value: true
+            value: options.isTTY ?? true
         });
         Object.defineProperty(this, "write", {
             enumerable: true,
             configurable: true,
             writable: true,
             value: (data) => {
-                this.emit('data', data);
+                this.data = data;
+                this.emit('readable');
+                // this.emit('data', data);
+            }
+        });
+        Object.defineProperty(this, "read", {
+            enumerable: true,
+            configurable: true,
+            writable: true,
+            value: () => {
+                const data = this.data;
+
+                this.data = null;
+
+                return data;
             }
         });
+        this.data = null;
     }
     setEncoding() {
         // Do nothing
@@ -98,6 +113,12 @@ class Stdin extends EventEmitter {
     pause() {
         // Do nothing
     }
+    ref() {
+        // Do nothing
+    }
+    unref() {
+        // Do nothing
+    }
 }
 const instances = [];
 export const render = (tree) => {
